sources/core/objects/Drinks.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Drinks.h"
+
+Drinks::Drinks(quint32 id) : Item(id)
+{
+    name = "BaseDrink";
+    SetSprite("icons/drinks.dmi");
+    full_ = false;
+}
+void Drinks::Drink(IdPtr<Human> human1, IdPtr<Human> human2)
+{
+    if(!full_)
+    {
+        full_ = true;
+        if(human1 == human2)
+        {
+            GetGame().GetChat().PostTextFor(
+            "You swallow a gulp of " + name + ".", human1->GetId());
```

Comment:
1. Extra 4 spaces
2. It is better to use `QString` `arg`s here - much more convenient than concatenation via `+`.